src/Module/Conversation/Network.php

```diff
@@ -340,7 +338,12 @@ protected static function parseRequest(array $parameters, array $get)
 			case 'uriid':
 				self::$max_id = $get['last_uriid'] ?? self::$max_id;
 				break;
+			default:
+				self::$order = 'commented';
+				self::$max_id = $get['last_commented'] ?? self::$max_id;
```

Comment:
Please remove the `case 'commented':` block since this does the same thing.

Reply:
Done